cloud/team/team.go

```diff
@@ -31,6 +32,11 @@ var (
 	teamPagnationLimit          = 100
 )
 
+func confirmOperation() {
+	fmt.Printf("This is an IDP-managed team. Please avoid making changes in Astro whenever possible. \n%s to continue the operation or %s to abort", ansi.Green("Press Enter"), ansi.Red("^C"))
+	fmt.Scanln()
```

Comment:
nit: from a consistency point of view: we generally use `input.Confirm` method for any user confirmation in CLI, which internally is y/n (yes/no) instead of enter/^C

Reply:
sound good, let me switch over

Reply:
fixed